src/tasks/run_closed_loop.py

```diff
@@ -55,15 +82,45 @@ def run():
     )
     task_params = _build_param_from_arg(args.task_settings_path, SETTINGS_PATH_PARAM)
 
+    logger.info("Starting modules")
     encoder = subprocess.Popen(["encoder"] + nds_params)
     ephys = subprocess.Popen(["ephys_generator"] + nds_params)
     decoder = subprocess.Popen(["decoder"] + decoder_params)
-    center_out_reach = subprocess.Popen(["center_out_reach"] + task_params)
 
-    center_out_reach.wait()
-    encoder.kill()
-    ephys.kill()
-    decoder.kill()
+    with tempfile.TemporaryDirectory() as temp_dir:
+        control_file_path = os.path.join(temp_dir, "center_out_reach_control_file")
+        open(control_file_path, "x")
```

Comment:
Actually, at this point, I'm sure that the file doesn't exist because the directory is unique and new, and then, the opening with the 'x' mode creates a new file without opening it for reading and writing; this is going to be done in the next steps in the code (and in the subprocess)